### ISSUE TITLE
[ BUG ] New-FalconIoc 'Expiration' RegEx error

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#156]
**Describe the bug**
When using `New-FalconIoc`, a timestamp of `2022-01-31T00:00:00Z` is not accepted.

**To Reproduce**
`New-FalconIoc ... -Expiration '2022-01-31T00:00:00Z'`

**Expected behavior**
Either timestamp format (`2022-01-31` or `2022-01-31T00:00:00Z`) should be accepted.

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1
 - PSFalcon: 2.1.6

**Comment by bk-cs** [comment]
This is due to a typo in the `ValidatePattern` property of the `Expiration` parameter.

Incorrect:
```
[ValidatePattern('^(\d{4}-\d{2}-\d{2}|\d{4}-\d{2}-\d{2}T\d{2}:\d{2}\d{2}Z)$')]
```

Correct:
```
[ValidatePattern('^(\d{4}-\d{2}-\d{2}|\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z)$')]
```

**Comment by bk-cs** [comment]
This issue has been fixed for the v2.1.7 release. To fix it yourself before the release, replace `Public\ioc.ps1` with the contents of this file: https://raw.githubusercontent.com/CrowdStrike/psfalcon/master/Public/ioc.ps1

**Comment by bk-cs** [comment]
Closing due to [v2.1.7 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.1.7).
